frontend/src/components/nodes/ValidatorMetadataRow.tsx

```diff
@@ -47,7 +47,7 @@ class ValidatorMetadataRow extends PureComponent<Props> {
                       <Col className="validator-nodes-text">
                         <a
                           href={
-                            poolWebsite.startsWith("http")
+                            poolWebsite?.startsWith("http")
```

Comment:
I believe `?` isn't necessary because of precheck over [here](https://github.com/near/near-explorer/pull/714/commits/34f1cfcc00df5c21dd4387ff387f3fa45ce2cad7#diff-263c463b37e6c396f67ea639ff4f6738b2590b36303806d0bfdc53a5f7304506R37). @IsQiao WDYT?

Reply:
yes, it's not necessary, only for robustness. i can remove it, so do I need to do this?

Reply:
Yes, please

Reply:
Fixed